### PR TITLE
English: Hosting & Security

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Translates Hosting & Security

### CODE REVIEW

File: source/localizable/hosting-security.en.erb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title:
+  nl: Hosting en Security
+  en: Hosting and Security
+description:
+  nl: Defacto biedt de mogelijkheid om CAPP LMS veilig en met betrouwbare security te hosten. Updates worden door Defacto geïnstalleerd en bijgewerkt.
```
Comment (fatboypunk) [inline source/localizable/hosting-security.en.erb]: biedt de mogelijkheid

not true, we don't offer the possibility we just do this

Comment (RonaldDijkstra) [inline source/localizable/hosting-security.en.erb]: True that! Nice one

File: source/localizable/hosting/_hosting.en.md.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+- CAPP LMS offers Standard Single Sign On (SSO-integration)
+- As old school software company, we know all about the technical ins and outs. This makes us a valued partner for every IT-organisation. You will not experience IT-related problems when working with us and our software.
+- CAPP is in compliance with the IT-security regulations for web applications – part 1, National Cyber Security Centre, Ministry of Security and Justice
+- CAPP includes a standard HR-Importer that can establish integrations with all well-known national why national for an EN website? and international HR-systems
+- If desired, the personal data of employees that is stored in HR-systems will be read into CAPP every night. For this, Defacto uses the SSH File Transfer Protocol (sFTP). In contrast to FTP, the data is encrypted while being transmitted, which makes the process very secure.
+- CAPP facilitates numerous possibilities for integration with (quality-)registers, assessment systems and suppliers of e-learning material.
+- CAPP is an open system in which you access e-learning material of all well-known e-learning suppliers.
```
Comment (sn3p) [inline source/localizable/hosting/_hosting.en.md.erb]: E-learning (for consistency)

File: source/localizable/hosting/_hosting.en.md.erb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+- CAPP LMS is completely web based and is accessible anytime, anywhere on any device.
+- CAPP LMS offers Standard Single Sign On (SSO-integration)
+- As old school software company, we know all about the technical ins and outs. This makes us a valued partner for every IT-organisation. You will not experience IT-related problems when working with us and our software.
+- CAPP is in compliance with the IT-security regulations for web applications – part 1, National Cyber Security Centre, Ministry of Security and Justice
+- CAPP includes a standard HR-Importer that can establish integrations with all well-known national why national for an EN website? and international HR-systems
+- If desired, the personal data of employees that is stored in HR-systems will be read into CAPP every night. For this, Defacto uses the SSH File Transfer Protocol (sFTP). In contrast to FTP, the data is encrypted while being transmitted, which makes the process very secure.
+- CAPP facilitates numerous possibilities for integration with (quality-)registers, assessment systems and suppliers of e-learning material.
```
Comment (sn3p) [inline source/localizable/hosting/_hosting.en.md.erb]: E-learning (for consistency)

File: source/localizable/hosting/_hosting.en.md.erb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+Defacto is hosting CAPP LMS in the most safe and reliable manner. Updates are being installed and reviewed by Defacto itself. We can only offer on-premise delivery models to a very limited extent.
+
+- CAPP LMS is completely web based and is accessible anytime, anywhere on any device.
+- CAPP LMS offers Standard Single Sign On (SSO-integration)
+- As old school software company, we know all about the technical ins and outs. This makes us a valued partner for every IT-organisation. You will not experience IT-related problems when working with us and our software.
```
Comment (sn3p) [inline source/localizable/hosting/_hosting.en.md.erb]: `As an old school ...`

Also `old school` doesn't sound positive in software terms, `experienced` probably fits better here?

**Pull request merged**